docs/index.md

```diff
@@ -11,15 +11,40 @@ This is the documentation for the `cfgov-refresh` project, a redesign of the [ww
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
+- [Apache]() - Version? Which project exactly? https://apache.org/index.html#projects-list
 - [Homebrew](https://brew.sh) - package manager for installing system software on OSX
-- Python and PIP (Python package manager)
-- [Jinja templates](http://jinja.pocoo.org) for front-end rendering
-- [Wagtail CMS](https://wagtail.io) for content administration
-- Dependencies, listed below
+- Python 2.7 and PIP (Python package manager)
+- [Jinja2 templates](http://jinja.pocoo.org/docs/2.10/) for front-end rendering. See [`requirements/libraries.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/libraries.txt) for version.
+- [Wagtail CMS](https://wagtail.io) for content administration. See [`requirements/wagtail.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/wagtail.txt) for version.
+- [PostgreSQL](https://www.postgresql.org/) and [Psycopg](http://initd.org/psycopg/) for our database. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for version.
+- Additional dependencies, listed below
 
-# Dependencies
+# Additional dependencies
 
 - [Elasticsearch](https://www.elastic.co):
   Used for full-text search capabilities and content indexing.
-- [Node](http://nodejs.org) and [yarn](https://yarnpkg.com/):
-  Used for downloading and managing front-end dependencies and assets.
+- [Node 8](http://nodejs.org) and [yarn](https://yarnpkg.com/):
+  Used for downloading and managing front-end dependencies and assets. Front-end dependencies are listed in the project's [package.json](https://github.com/cfpb/cfgov-refresh/blob/master/package.json) file.
+- [Gulp 4](https://gulpjs.com/)
+- [virtualenv](https://virtualenv.pypa.io/en/stable/)
+- [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
+
+# Versions
+
+Versions for most front-end packages are kept updated in the project's [package.json](https://github.com/cfpb/cfgov-refresh/blob/master/package.json) file.
+
+Versions for back-end software including Python, Jinja, Wagtail, PostgreSQL, etc. are kept in the project's requirements files:
+https://github.com/cfpb/cfgov-refresh/tree/master/requirements
+
+- `base.txt`: shortcut for `django.txt` + `wagtail.txt` + `libraries.txt`
+- `deployment.txt`: requirements for deployment, includes `base.txt` and `postgres.txt` and a New Relic library which we don't install anywhere else.
+- `django.txt`: specifies Django version. In its own file to make it easier to test multiple versions. The file is used when running the site, but by having it separate we can test against other versions of Django by excluding this file.
```

Comment:
```suggestion
- `django.txt`: specifies the Django version. The file is used when running the site, but by having it separate we can test against other versions of Django by excluding this file.
```

I'm suggesting removing one of the sentences here — both of them together seem unnecessary and tautological. I'm not sure if one is preferable to the other though.